scripts/index.ts

```diff
@@ -8,7 +8,7 @@
 
 const [, , ...steps] = process.argv;
 const {
-  CI_PULL_REQUEST,
+  CIRCLE_PR_NUMBER,
```

Comment:
Is this env var set by CircleCI?

Reply:
Yes. Forgot to mention that `CIRCLE_PR_NUMBER` is always set for PRs, but `CI_PULL_REQUEST` is unset when the CI is re-run through the CircleCI UI.